### PR TITLE
ApiServer and A0BlockHeader fix

### PR DESCRIPTION
**Pull request by AionJayT**
## Notice

It is not allowed to submit your PR to the master branch directly, please submit your PR to the master-pre-merge branch.

## Description

Please include a brief summary of the change that this pull request proposes. Include any relevant motivation and context. List any dependencies required for this change.

1. Fix load error happens in IntelliJ due to missing module dependency.
2, Fix A0BlockHeader toJson method output inconsistency(only effect the dashboard API);

-

Fixes Issue # .

## Type of change

Insert **x** into the following checkboxes to confirm (eg. [x]):
- [x] Bug fix.
- [ ] New feature.
- [ ] Enhancement.
- [x] Unit test.
- [ ] Breaking change (a fix or feature that causes existing functionality to not work as expected).
- [ ] Requires documentation update.

## Testing

Please describe the tests you used to validate this pull request. Provide any relevant details for test configurations as well as any instructions to reproduce these results.

- Updated the test case in A0BlockHeaderTest

## Verification

Insert **x** into the following checkboxes to confirm (eg. [x]):
- [x] I have self-reviewed my own code and conformed to the style guidelines of this project.
- [x] New and existing tests pass locally with my changes.
- [x] I have added tests for my fix or feature.
- [ ] I have made appropriate changes to the corresponding documentation.
- [x] My code generates no new warnings.
- [x] Any dependent changes have been made.

### CODE REVIEW

File: modAionImpl/test/org/aion/zero/impl/types/A0BlockHeaderTest.java [hunk 1]
```diff
@@ -129,6 +132,7 @@ public void testBlockHeaderFromUnsafeSource() throws Exception {
         assertThat(header.getSolution()).isEqualTo(new byte[1408]);
         assertThat(header.getNonce()).isEqualTo(ByteUtil.EMPTY_WORD);
         assertThat(header.getDifficulty()).isEqualTo(ByteUtil.EMPTY_HALFWORD);
+        assertThat(header.getVersion() == (byte) 0x01);
```
Comment (aion-kelvin) [inline modAionImpl/test/org/aion/zero/impl/types/A0BlockHeaderTest.java]: style: assertThat(header.getVersion()).isEqualTo((byte) 0x01);

File: modAionImpl/test/org/aion/zero/impl/types/A0BlockHeaderTest.java [hunk 2]
```diff
@@ -165,6 +170,7 @@ public void testBlockHeaderFromRLP() throws Exception {
         assertThat(reconstructed.getParentHash()).isEqualTo(header.getParentHash());
         assertThat(reconstructed.getNonce()).isEqualTo(header.getNonce());
         assertThat(reconstructed.getDifficulty()).isEqualTo(header.getDifficulty());
+        assertThat(reconstructed.getVersion() == header.getVersion());
```
Comment (aion-kelvin) [inline modAionImpl/test/org/aion/zero/impl/types/A0BlockHeaderTest.java]: style: assertThat( ... ).isEqualTo( ... );

File: modAionImpl/test/org/aion/zero/impl/types/A0BlockHeaderTest.java (hunk 1 above)
Comment (AionJayT) [reply modAionImpl/test/org/aion/zero/impl/types/A0BlockHeaderTest.java]: the getVersion is returning byte, doesn't have the isEqualTo method.

File: modAionImpl/test/org/aion/zero/impl/types/A0BlockHeaderTest.java (hunk 2 above)
Comment (AionJayT) [reply modAionImpl/test/org/aion/zero/impl/types/A0BlockHeaderTest.java]: some here.

**Pull request merged**